### PR TITLE
update proto to include verified queries; lint

### PR DESCRIPTION
**Pull request by sfc-gh-rehuang**
This PR mainly introduced VerifiedQuery into proto. 
Add Column to proto for backward compatibility for some utils (not showing to end user)

### CODE REVIEW

File: semantic_model_generator/protos/semantic_model.proto [hunk 1]
```diff
@@ -149,4 +150,68 @@ message SemanticModel {
...
+    // The correct SQL query for answering the question.
+    string sql = 4;
+    // Timestamp at which the query was last verified - measures in seconds since epoch, in UTC.
+    int64 verified_at = 5 [(optional) = true];
+    // Name of the person who verified this query.
+    string verified_by = 6 [(optional) = true];
+}
+
+// VerifiedQueryRepository is a simply a collection of verified queries.
+message VerifiedQueryRepository {
```
Comment (sfc-gh-nsehrawat) [inline semantic_model_generator/protos/semantic_model.proto:215]: This is needed for having VQR as a separate file.  Should we drop this for now and rather just add a `repeated VerifiedQuery` field in `SemanticModel` proto?

File: semantic_model_generator/protos/semantic_model.proto [hunk 2]
```diff
@@ -149,4 +150,68 @@ message SemanticModel {
...
+    // Sample values of this column.
+    repeated string sample_values = 9 [(optional) = true];
+}
+
+
+// VerifiedQuery represents a (question, sql) pair that has been manually verified
+// (e.g. by an analyst) to be correct.
+message VerifiedQuery {
+    // A name for this verified query. Mainly used for display purposes.
+    string name = 1;
```
Comment (sfc-gh-nsehrawat) [inline semantic_model_generator/protos/semantic_model.proto:201]: Should we make this optional as well?

Comment (sfc-gh-rehuang) [reply semantic_model_generator/protos/semantic_model.proto]: good point, updated!

File: semantic_model_generator/protos/semantic_model.proto (hunk 1 above)
Comment (sfc-gh-rehuang) [reply semantic_model_generator/protos/semantic_model.proto]: removed!

**Review by sfc-gh-nsehrawat — APPROVED** [review]

**Pull request merged**